opendigitaleducation/entcore
# [EDIFICE] Réindexation des ressources par leur état d'ingestion

# Description

Permettre de réindexer les ressources par leur état d'ingestion.

## Fixes

wb-2243

## Type of change

Please check options that are relevant.


## Which packages changed?

Please check the name of the package you changed


## Tests

1. Create a blog
2. In mongo, set the `ingest_job_state` to KO and modify the field `description`
3. With an ADMC, go to http://localhost:8090/explorer/reindex/blog/blog?states=KO
4. Verify that the return of the call says that one message was concerned
5. Verify in mongo that `ingest_job_state` is set to OK
6. Verify in opensearch that the corresponding document has a `description` matching the one in the database

# Reminder

- Security flaws
- Performance impacts (think bulk !)
- Unit tests were replayed
- Unit tests were added and/or changed
- I have updated the reminder for the version including my modifications
